src/docs/tutorials/set-up-a-tailscale-subnet-router.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Set up a Tailscale Subnet Router
+---
+
+## What is a Subnet router?
+
+> A subnet router is a device within your tailnet that you use as a gateway that advertises routes for other devices that you want to connect to your tailnet without installing the Tailscale client.
+
+*Source: <a href="https://tailscale.com/kb/1019/subnets" target="_blank">Subnet routers</a> Via Tailscale's Documentation*
+
+In the context of Railway, The "other devices" are the services within a project.
+
+## About this Tutorial
+
+The goal of this tutorial is to enable us to connect to our production Postgres database without ever needing to expose it publicly.
+
+Since Railway doesn't currently offer a native way to access the <a href="https://docs.railway.app/reference/private-networking" target="_blank">private network</a> from our local environment, we can use Tailscale as a subnet router to accomplish this.
+
+Deploying Tailscale as a subnet router into our project means that we can access the `railway.internal` private domains from any device connected to our tailnet.
+
+This tutorial aims to provide a simple step-by-step guide on setting up everything needed so that we can access the private domains of our services.
+
+**Objectives**
+
+In this tutorial, you'll learn how to do the following: -
+
+- Generate an Auth Key.
+- Set up split DNS.
+- Deploy the Tailscale Subnet Router template.
+- Approve the private network subnet.
+- (Bonus) Connect to Postgres locally via the private domain.
+
+**Prerequisites**
+
```

Comment:
Maybe?

"This guide assumes you are familiar with the concepts of Private Network, for a quick explainer check out <link> and <link>"